packages/flutter/test/material/filled_button_test.dart

```diff
@@ -1845,6 +1846,7 @@ void main() {
       count += 1;
     }
     final MaterialStatesController controller = MaterialStatesController();
+    addTearDown(() => controller.dispose());
```

Comment:
I missed this. There are still a few occurrences in the pr that can be made `addTearDown(() => [name].dispose())` -> `addTearDown([name].dispose)` can we change these?

similar for overlay entry disposal `addTearDown(entry..remove..dispose)`.

Reply:
It does not work for entry, because entry is late and is not initialized at this moment.
Will fix others.